index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    <header class="first">
+      <nav class="menu">
+        <ul>
+          <li><a href="#" class="logo">Logo</a></li>
+          <li><a href="#"><img src="./images/union.svg" alt="Hamburger logo"></a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="second">
+      <h1>I’m Yersel<br>
+        Glad to see you!
+      </h1>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h2>LET'S CONNECT</h2>
+      <div class="icons">
+        <ul>
+          <li><a href="#"><img src="./images/twitter.svg" alt="Twitter logo"></a></li>
+          <li><a href="#"><img src="./images/Linkedin.svg" alt="Linkedin logo"></a></li>
+          <li><a href="#"><img src="./images/m.svg" alt="m logo"></a></li>
+          <li><a href="#"><img src="./images/github.svg" alt="github logo"></a></li>
+          <li><a href="#"><img src="./images/hands.svg" alt="hands logo"></a></li>
+        </ul>
+
+      </div>
+      
+    </main>
+  </div>
+  
+</body>
+</html>
```

Comment:
- [ ] OPTIONAL: I suggest trying to follow the correct paragraph spacing, as shown in the design:

The Figma:

![image](https://user-images.githubusercontent.com/57421823/148336567-2d117376-2158-47a1-9efe-a66ba5ec0232.png)

Your page:

![image](https://user-images.githubusercontent.com/57421823/148336826-c9c2f8da-99e2-474a-b10f-f4133a0fdc74.png)

If you do this, your portfolio will be a lot more consistent and professional 😃